src/main/java/org/commcare/formplayer/repo/impl/PostgresFormSessionRepo.java

```diff
@@ -54,7 +54,11 @@ public void purgeFormSessions() {
             String deleteQuery = replaceTableName(
                     "delete from %s where custom_safe_cast(dateopened, '2011-01-01'::timestamp) < NOW() - INTERVAL '7 days';"
             );
-            this.jdbcTemplate.execute(deleteQuery);
+            log.info("Beginning state form session purge");
+            long start = System.currentTimeMillis();
+            int deletedRows = this.jdbcTemplate.update(deleteQuery);
+            long elapsed = (System.currentTimeMillis() - start) / 1000;
+            log.info(String.format("Purged %d stale form sessions in %d ms", deletedRows, elapsed));
```

Comment:
looks like elapsed is in seconds (Millis / 1000), so if I'm reading it right either that line should get rid of the `/ 1000` or this line should have `s` instead of `ms`, right?

Reply:
oh, whoops, I meant to get rid of the divide, thanks for catching.